specification/storagecache/resource-manager/readme.ruby.md

```diff
@@ -0,0 +1,19 @@
+## Ruby
+
+These settings apply only when `--ruby` is specified on the command line.
+
+```yaml
+package-name: azure_mgmt_storagecache
+package-version: 2019-08-01
+azure-arm: true
+```
+
+### Tag: package-2019-12-01 and ruby
+
+These settings apply only when `--tag=package-2019-08-01 --ruby` is specified on the command line.
```

Comment:
In some files, you use tag of 'tag=package-2019-08' and in other files you use '--tag=package-2019-08-01', is there any reason to use different tags?